com.unity.formats.fbx/Editor/ExportModelSettings.cs

```diff
@@ -114,9 +114,18 @@ public override void OnInspectorGUI ()
             EditorGUI.EndDisabledGroup();
             GUILayout.EndHorizontal();
 
-            exportSettings.SetKeepInstances(EditorGUILayout.Toggle(new GUIContent("Keep Instances", "If enabled, instances will be preserved as instances in the FBX file. This can cause issues with e.g. Blender if different instances have different materials assigned."), exportSettings.KeepInstances));
-            exportSettings.SetEmbedTextures(EditorGUILayout.Toggle(new GUIContent("Embed Textures", "If enabled, textures are embedded into the resulting FBX file instead of referenced."), exportSettings.EmbedTextures));
+            GUILayout.BeginHorizontal();
```

Comment:
indentation didn't match the other export options (more noticeable in the Project Settings)